integration_tests/features/execute_sql_function.feature

```diff
@@ -0,0 +1,16 @@
+@dbtv1
+Feature: `execute_sql` function
+  Background: Project Setup
+    Given the project 009_execute_sql_function
```

Comment:
we have a functions project, do we need a new one because it is v1 only?

Reply:
Yes. I thought about making it a part of functions project, but in the end this seemed like the tidiest solution, as we we don't have to worry about model selection in other features.